assets/css/dialogue.css

```
@@ -0,0 +1,34 @@
+/* dialogue voices (voice shortcode) — colors come from theme vars so they survive the per-deploy theme roulette */
+
+.voice {
+    --voice-color: var(--color-primary);
+    border-left: 3px solid var(--voice-color);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)